api/features/tests/test_views.py

```diff
@@ -656,15 +660,18 @@ def test_get_flags_hide_sensitive_data(api_client, environment, feature):
     # When
     api_client.credentials(HTTP_X_ENVIRONMENT_KEY=environment.api_key)
     response = api_client.get(url)
+    fs_sensitive_fields = SDKFeatureStateSerializer.sensitive_fields
+    feature_sensitive_fields = SDKFeatureSerializer.sensitive_fields
```

Comment:
I'd prefer those fields explicitly tested here. E.g. if we change `SDKFeatureStateSerializer.sensitive_fields` to `()` the feature will be broken but the test won't.